src/TeamleaderDotNet/Utils/UnixTimestampExtensions.cs

```diff
@@ -44,7 +44,7 @@ public static DateTime UnixTimeToDateTime(this int unixtime)
         /// <returns>Returns a DateTime object that represents value of the Unix time.</returns>
         public static DateTime UnixTimeToDateTime(this string unixtime)
         {
-            return UnixTimeToDateTime(long.Parse(unixtime.ToString()));
+            return !string.IsNullOrEmpty(unixtime) ? UnixTimeToDateTime(long.Parse(unixtime)) : DateTime.MinValue;

```

Comment:
@jeroenverh
The problem is however, if teamleader returns an empty unixdatetime, the UnixDateConverter does not achieve in converting the value to a datetime. If you want that behaviour, you need to modify to a nullable datetime like a did with deals. I didn't modify exisiting classes. You can do that but than you need to create a new major version, which is not needed for this pull request. Can you please create a new nuget package? Thanks!